src/javascript/utils.js

```diff
@@ -49,7 +49,7 @@ function is(variable, type) {
 }
 
 /**
- * Merge objects together. Will remove 'falsy' values.
+ * Merge objects together. Will remove undefined and null values.
```

Comment:
I've updated this to reflect what the code is actually checking for.  But this isn't quite right either - what it's really doing is ignoring undefined and null values in the object being merged in.  So there's no way to delete an existing key right now, while from this I assumed that using a falsey value would unset the key from the merged object...